python/astro_metadata_translator/translators/decam.py

```diff
@@ -248,3 +251,44 @@ def to_detector_name(self):
         # Docstring will be inherited. Property defined in properties.py
         name = self.to_detector_unique_name()
         return name[1:]
+
+    @classmethod
+    def fix_header(cls, header, obsid, filename=None):
+        """Fix DECam headers.
+
+        Parameters
+        ----------
+        header : `dict`
+            The header to update.  Updates are in place.
+        obsid : `str`
+            Unique observation identifier associated with this header.
+            Will always be provided.
+        filename : `str`, optional
+            Filename associated with this header. May not be set since headers
+            can be fixed independently of any filename being known.
+
+        Returns
+        -------
+        modified = `bool`
+            Returns `True` if the header was updated.
+
+        Notes
+        -----
+        Fixes the following issues:
+        * If OBSTYPE contains "zero" or "bias",
+          update the FILTER keyword to "solid plate 0.0 0.0".
+
+        Corrections are reported as debug level log messages.
+        """
+        modified = False
+
+        obsid = header.get("OBSID", "unknown")
+        obstype = header.get("OBSTYPE", "unknown")
```

Comment:
Maybe add the `.lower()` on the end of this? (I think it might already be lower case) Then you wouldn't need to specify it below.

Reply:
I prefer to keep `obstype` defined exactly as the header has it so I can use it in the log message